### PR TITLE
README.md in Japanese.

### PR DESCRIPTION
**Pull request by kaznak**
Hello.

I have translated README.md into Japanese.
This will help elm-lang to spread in Japan.
Would you merge it if all right?

thank you.

### CODE REVIEW

**Comment by igrep** [comment]
Great!

File: README.ja.md [hunk 1]
```diff
@@ -0,0 +1,778 @@
...
+  1. [カウンタ](http://evancz.github.io/elm-architecture-tutorial/examples/1.html)
+  2. [二つのカウンタ](http://evancz.github.io/elm-architecture-tutorial/examples/2.html)
+  3. [カウンタのリスト](http://evancz.github.io/elm-architecture-tutorial/examples/3.html)
+  4. [カウンタのリスト(別の実装)](http://evancz.github.io/elm-architecture-tutorial/examples/4.html)
+  5. [GIF 取得器](http://evancz.github.io/elm-architecture-tutorial/examples/5.html)
+  6. [二つの GIF 取得器](http://evancz.github.io/elm-architecture-tutorial/examples/6.html)
+  7. [GIT 取得器のリスト](http://evancz.github.io/elm-architecture-tutorial/examples/7.html)
+  8. [二つのアニメーションする四角形](http://evancz.github.io/elm-architecture-tutorial/examples/8.html)
+
+このチュートリアルは実際助ける!例 7 と 8 を超簡単に作るために必要な概念と考え方を明らかにする。 Elm の基礎に時間を費やすのはその価値があるよ!
```
Comment (igrep) [inline README.ja.md]: How about 「実際助ける」 => 「本当に役に立つ」
http://ejje.weblio.jp/content/help

**Comment by igrep** [comment]
I've read just by the Example 1, looks good!

File: README.ja.md (hunk 1 above)
Comment (kaznak) [inline README.ja.md]: sounds good! I have put your one into my repo.

**Comment by evancz** [comment]
Very cool, thank you!

I'll merge for now, but I'm not sure what to do if the text ever changes. I guess we'll find out!

**Comment by kaznak** [comment]
You too!

**Pull request merged**